### PR TITLE
fix: wrong property names "ingerssHost" modify to "ingressHost"

### PR DESCRIPTION
**Pull request by magic-hya**
Fix  wrong property names "ingerssHost" modify to "ingressHost" in values-template.yaml

When I used version 1.7.0, I found this problem. Now 1.9.0 still exists. So I submit this fix

### CODE REVIEW

File: helm-charts/FATE-Serving/values-template.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ servingProxy:
   type: {{ .type }}
   nodePort: {{ .nodePort }}
   loadBalancerIP: {{ .loadBalancerIP }}
-  ingerssHost: {{ .ingressHost | default (printf "%s.serving-admin.example.com" $partyId) }}
+  ingressHost: {{ .ingressHost | default (printf "%s.serving-admin.example.com" $partyId) }}
```
Comment (JingChen23) [inline helm-charts/FATE-Serving/values-template.yaml:73]: Instead of changing this, you can remove this.

**Review by JingChen23 — CHANGES_REQUESTED** [review]
Hi @magic-hya , thanks for the fix! Two things:
1. I just checked the helm chart, there is no place using this variable. So let's remove it. But still thank you for finding the typo.
2. You need to raise a PR to the branch `develop-1.9.0`. `master` cannot be changed now. When 1.9.0 is released, everything will be merged to master in one big PR.

**Comment by JingChen23** [comment]
Also, please follow the `Details` to resolve your DOC issue.

**Comment by magic-hya** [comment]
I've tried to modify it, but there are more mistakes. I'll open a new PR tomorrow，thank！

**Pull request closed without merging**